src/download.html

```diff
@@ -208,7 +208,8 @@ <h1 class="header">Thanks for downloading!</h1>
 						<span class="text">Debian/Ubuntu</span>
 					</label>
 					<div class="tabdiv">
-						<p>For Debian, Ubuntu, Mint, and other distros using the apt package manager, you'll need to run the following commands to add our repository.</p>
+						<p>For Distributions utilising the deb format, you'll need to run the following commands to add our repository.</p>
```

Comment:
"Distributions" doesn't need to be capitalized, and you can say "deb package format" or "deb packages".